daemon/src/model/cfd.rs

```diff
@@ -2361,6 +2373,140 @@ mod tests {
         assert_eq!(funding_fee.rate, funding_rate);
     }
 
+    #[test]
+    fn ensure_collaborative_settlement_takes_rollover_fees_into_account() {
```

Comment:
Split these into multiple (i.e. one assertion per test).

Reply:
Could we do that in future iterations once we have more tests? 
I think it really helps grouping them to see the behavior of the specific scenario in comparison (i.e. all of these tests are related to the number of rollovers). If I split this and then add more tests on different scenarios I think this will become messy and hard to see for the reader what is related. 

Is there a possibility for grouping tests together in rust? use a module? 🤔